llama_index/evaluation/__init__.py

```diff
@@ -1,12 +1,14 @@
 """Evaluation modules."""
 
-from llama_index.evaluation.base import ResponseEvaluator, QueryResponseEvaluator
+from llama_index.evaluation.base import QueryResponseEvaluator, ResponseEvaluator
 from llama_index.evaluation.dataset_generation import DatasetGenerator
 from llama_index.evaluation.guideline_eval import GuidelineEvaluator
+from llama_index.evaluation.query_engine_eval import QueryEngineEvaluator
 
 __all__ = [
     "ResponseEvaluator",
     "QueryResponseEvaluator",
     "DatasetGenerator",
     "GuidelineEvaluator",
+    "QueryEngineEvaluator",
```

Comment:
nit: if you name this `*Evaluator`, I'd expect it to fit the `BaseEvaluator` interface. Otherwise, we should probably:
1. refactor the `BaseEvaluator` interface
2. implement this as a different concept

Reply:
yeah I've been thinking about the refactoring too. Currently `BaseEvaluator` does not have `evaluate()` as an abstract method which is why I thought it might not be important. I'll think more about it.